torch/csrc/dynamo/eval_frame.c

```diff
@@ -854,7 +974,7 @@ static PyObject* skip_code(PyObject* dummy, PyObject* obj) {
     PyErr_SetString(PyExc_TypeError, "expected a code object");
     return NULL;
   }
-  set_cache_entry((PyCodeObject*)obj, SKIP_CODE);
+  set_extra_state((PyCodeObject*)obj, SKIP_CODE);
```

Comment:
This seems to be a pre-existing problem, but you need to check if the original entry was non-NULL and deallocate it.